changelog/fragments/scorecard-parallel.yaml

```diff
@@ -1,6 +1,6 @@
 entries:
   - description: >
-      Changes the scorecard configuration format to include a new top-level
+      Changed the scorecard configuration format to include a new top-level
```

Comment:
We should combine and updated this fragment with the scorecard component config PR fragment. Don't have to do that now though. I think there will be a bunch of this type of consolidation in the release prep PR.